Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ruby:2.3.0
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev libgmp3-dev
+
+# gpg keys listed at https://github.com/nodejs/node
+RUN set -ex \
+  && for key in \
+    9554F04D7259F04124DE6B476D5A82AC7E37093B \
```

Comment:
can you explain this part for me? Is this a key? can we manage this differently?